eta-meeting-organizer-backend/src/main/java/hu/flowacademy/meetingorganizer/service/MeetingRoomService.java

```diff
@@ -0,0 +1,39 @@
+package hu.flowacademy.meetingorganizer.service;
+
+import hu.flowacademy.meetingorganizer.persistence.model.MeetingRoom;
+import hu.flowacademy.meetingorganizer.persistence.repository.MeetingRoomRepository;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+
+@Service
+@AllArgsConstructor
+@Transactional
+public class MeetingRoomService {
+    private MeetingRoomRepository meetingRoomRepository;
+
+    public List<MeetingRoom> findAllMeetingRooms () {
+        return meetingRoomRepository.findAll();
+    }
+
+    public MeetingRoom findOneMeetingRoomById (Long id) {
```

Comment:
[minor] A metódus neve lehetne csak simán `findById()`, kicsit redundáns a MeetingRoom (mivel ugye alapból a service neve is erre utal) és a One (mivel id, ezért csak egy lehet belőle legfeljebb).